modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/image/ImageDDMFormFieldTemplateContextContributor.java

```diff
@@ -277,9 +256,12 @@ private JSONObject _getValueJSONObject(String value) {
 	@Reference
 	private DLURLHelper _dlURLHelper;
 
-	private final List<ImageDDMFormFieldItemSelectorCriterionContributor>
-		_imageDDMFormFieldItemSelectorCriterionContributors =
-			new CopyOnWriteArrayList<>();
+	@Reference(
+		cardinality = ReferenceCardinality.MULTIPLE,
```

Comment:
@W2jx As Tina said in the other PR she closed, this is not needed.

Looking at the JavaDoc for `@Reference`: https://docs.osgi.org/javadoc/r6/cmpn/org/osgi/service/component/annotations/Reference.html

### cardinality

If not specified, the cardinality of this reference is based upon how this annotation is used:

- Annotated field - The cardinality is based on the type of the field. If the type is either java.util.Collection, java.util.List, or a subtype of java.util.Collection, the cardinality is [0..n](https://docs.osgi.org/javadoc/r6/cmpn/org/osgi/service/component/annotations/ReferenceCardinality.html#MULTIPLE). Otherwise the cardinality is [1..1](https://docs.osgi.org/javadoc/r6/cmpn/org/osgi/service/component/annotations/ReferenceCardinality.html#MANDATORY).

### policy

If not specified, the policy of this reference is based upon how this annotation is used:

- Annotated field - The policy is based on the modifiers of the field. If the field is declared volatile, the policy is [ReferencePolicy.DYNAMIC](https://docs.osgi.org/javadoc/r6/cmpn/org/osgi/service/component/annotations/ReferencePolicy.html#DYNAMIC). Otherwise the policy is [STATIC](https://docs.osgi.org/javadoc/r6/cmpn/org/osgi/service/component/annotations/ReferencePolicy.html#STATIC).

### policyOption

- If not specified, the [RELUCTANT](https://docs.osgi.org/javadoc/r6/cmpn/org/osgi/service/component/annotations/ReferencePolicyOption.html#RELUCTANT) reference policy option is used.

**Based on this doc,**

- you don't need to specify `cardinality` for a `List` field, if the original `cardinality` is `MULTIPLE`;
- you need to specify `option` as `DYNAMIC`, or make the field `volatile` only when the original reference has `policy = DYNAMIC`;
- you need to specify `policyOption` as `GREEDY` only when the original reference has `policyOption = GREEDY`

cc @tinatian @JanisZhang @slnn